### PR TITLE
Vp 163/bugfix/settings profile general information

### PR DESCRIPTION
**Pull request by AlexeYankov**
added modal for general infiormation (when avatar uploaded and you want delete profile photo. Added modal suggestion to approve changes)

### CODE REVIEW

File: locales/ru.json [hunk 1]
```diff
@@ -142,16 +148,16 @@
...
         "required": "Имя пользователя обязательно для заполнения"
       },
       "lastName": {
-        "matches": "Имя пользователя может содержать только буквенные символы",
-        "min": "Имя должно содержать минимум 1 символ",
-        "max": "Имя не должно превышать 50 символов",
-        "required": "Имя пользователя обязательно для заполнения"
+        "matches": "Фамилия пользователя может содержать только буквенные символы",
+        "min": "Фамилия должно содержать минимум 1 символ",
+        "max": "Фамилия не должно превышать 50 символов",
```
Comment (LovaVikasso) [inline locales/ru.json:159]: Фамилия не должна

File: locales/ru.json [hunk 2]
```diff
@@ -142,16 +148,16 @@
...
+        "required": "Фамилия пользователя обязательно для заполнения"
```
Comment (LovaVikasso) [inline locales/ru.json:160]: Фамилия обязательна

File: src/app/[locale]/settings-profile/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -30,12 +30,14 @@ const TabsDemo = () => {
   const [getUserProfile] = useLazyGetProfileQuery();
 
   const onSetUserAvatar = (e: ChangeEvent<HTMLInputElement>) => {
+    console.log(1);
```
Comment (LovaVikasso) [inline src/app/[locale]/settings-profile/Tabs/Tabs.tsx:33]: delete later?

File: locales/en.json [hunk 4]
```diff
@@ -104,6 +104,12 @@
       "question": "Do you really want to delete a Following",
       "btnYes": "Yes",
       "btnNo": "No"
+    },
+    "DeleteAvatarModal": {
+      "title": "Delete profile photo",
+      "question": "Do you really want delete uploaded profile photo",
```
Comment (LovaVikasso) [inline locales/en.json:110]: Question mark?

File: locales/ru.json [hunk 5]
```diff
@@ -104,6 +104,12 @@
       "question": "Вы действительно хотите удалить подписчика",
       "btnYes": "Да",
       "btnNo": "Нет"
+    },
+    "DeleteAvatarModal": {
+      "title": "Удалить фото профиля",
+      "question": "Вы действительно хотите удалить загруженное фото профиля",
```
Comment (LovaVikasso) [inline locales/ru.json:110]: Question mark?

**Review by LovaVikasso — APPROVED** [review]

File: locales/ru.json (hunk 1 above)
Comment (AlexeYankov) [reply locales/ru.json]: corrected

File: locales/ru.json (hunk 2 above)
Comment (AlexeYankov) [reply locales/ru.json]: corrected

File: src/app/[locale]/settings-profile/Tabs/Tabs.tsx (hunk 3 above)
Comment (AlexeYankov) [reply src/app/[locale]/settings-profile/Tabs/Tabs.tsx]: corrected

File: locales/en.json (hunk 4 above)
Comment (AlexeYankov) [reply locales/en.json:110]: it have to be in all questions or nowhere

File: locales/ru.json (hunk 5 above)
Comment (AlexeYankov) [reply locales/ru.json:110]: it have to be in all questions or nowhere

**Review by maskpa100 — APPROVED** [review]

**Pull request merged**